### PR TITLE
Removed obsolete rateMode boolean.

### PR DESCRIPTION
**Pull request by carturn**
This removes rateMode boolean, people should use PIDSourceType instead.
CustomPID makes this reasonable.

### CODE REVIEW

File: custom/sensors/CANEncoder.java [hunk 1]
```diff
@@ -88,10 +66,13 @@ public void setReverseDirection(boolean reverseDirection) {
 	}
 	
 	public double pidGet() {
-		if (rateMode) {
-			return getRate();
+		switch (pidSource) {
```
Comment (c0d3rman) [inline custom/sensors/CANEncoder.java]: Is a switch statement really necessary? PIDSourceType is an enum.

File: custom/sensors/CANEncoder.java [hunk 2]
```diff
@@ -12,58 +12,36 @@
 	private PIDSourceType pidSource;
```
Comment (c0d3rman) [inline custom/sensors/CANEncoder.java]: I don't like this name, but it's the one Encoder uses in wpilib, so OK.

File: custom/sensors/CANEncoder.java (hunk 1 above)
Comment (carturn) [inline custom/sensors/CANEncoder.java]: YES

File: custom/sensors/CANEncoder.java (hunk 2 above)
Comment (carturn) [inline custom/sensors/CANEncoder.java]: OK.

**Comment by c0d3rman** [comment]
:+1:

**Pull request merged**